packages/appeals-service-api/src/lib/get-address.js

```diff
@@ -0,0 +1,14 @@
+const getAddress = (siteAddress) =>
```

Comment:
Should this function have a specified response or error if address is empty?

Reply:
Probably ignore this comment. I can see that your tests are happy with empty responses.